setup.py

```diff
@@ -55,7 +55,6 @@ def find_version(*file_paths):
         'flask>=0.10.0',
         'lxml>=3.4.4',
         'spyne>=2.12.8',
-        'nose',
```

Comment:
Note: WOFpy can be used without `odm2api` (it can use data sources and databases other than ODM2). An odm2api pypi package was created in December, but it needs to be recreated to match a clearer odm2api release. `geoalchemy` (or rather `geoalchemy-odm2`) used to be a dependency for odm2api, but it's not any more, since ~ 6 months ago.